lotteryPowerBall.py

```diff
@@ -0,0 +1,70 @@
+import random
+AMOUNT_OF_MOST_OFTEN_NUM = 5
+
+
+#  changes number from 1 to 9 in format
+#  with 0 in the beginning (task requirement)
+def change_number_format(num):
+    if num in [1, 2, 3, 4, 5, 6, 7, 8, 9]:
+        num = f'0{num}'
+    return num
+
+
+# first 5 numbers need to be in range [1, 69]
+# the last one - is Power Ball - in range [1, 26]
+def create_lottery_numbers():
+    file = open('numbers.txt', 'w')
+    for i in range(100):
+        power_num = change_number_format(random.randint(1, 27))
+        for j in range(5):
+            num = change_number_format(random.randint(1, 70))
+            file.write(f'{num} ')
+        file.write(f'{power_num}\n')
+    file.close()
+
+
+#  create a dictionary with amount of each digits
+def set_of_digits_from_file():
+    file = open('numbers.txt', 'r')
+    list_of_digits = []
+    for line in file:
+        list_of_digits.extend(line.split())
+    file.close()
+    return {digit: list_of_digits.count(digit) for digit in list_of_digits}
```

Comment:
This is not very optimal - `list_of_digits.count(digit)` has to look at entire array, and you run it for every element of array. You can get the same result by looping through array only once